src/nupic/types/Sdr.hpp

```diff
@@ -230,12 +232,12 @@ class SparseDistributedRepresentation : public Serializable
     /**
      * @attribute dimensions A list of dimensions of the SDR.
      */
-    const vector<UInt> &dimensions = dimensions_;
+    const vector<UInt> &dimensions() const { return dimensions_; }
 
     /**
      * @attribute size The total number of boolean values in the SDR.
      */
-    const UInt &size = size_;
+    UInt size() const { return size_; }
```

Comment:
for some reason it seems size() breaks gtest compilation (sizeof(..somegteststuff(x)))